src/file_loader.js

```diff
@@ -19,10 +19,14 @@ const readDirSync = dir =>
       ),
       []);
 
-const fileLoader = (folderPath, options = { recursive: false }) => {
+const fileLoader = (folderPath, options = {}) => {
```

Comment:
Would it be possible for you to change the fileLoader's signature to use destructuring and default params. e.g - `fileLoader = ({ recursive = false, extensions = DEFAULT_EXTENSIONS } = { recursive: false, extensions: DEFAULT_EXTENSIONS }) => ...`